src/main/scala/com.snowplowanalytics.snowplow/scalatracker/Tracker.scala

```diff
@@ -427,6 +430,44 @@ class Tracker(emitters: Seq[TEmitter], namespace: String, appId: String, encodeB
         System.err.println(s"Failed to retrieve context: $message")
         send(payload)
     }
+ 
+  /** 
+   *  Sends a Snowplow Event when error is non fatal.
+   *  
+   *  @param error The throwable
+   */
+  def trackError(error: Throwable): Unit = error match {
+    case NonFatal(e) =>
```

Comment:
I actually now think that `NonFatal` check should be removed. As I said earlier, user for some reasons passed here exactly this throwable and he/she wants it to be tracked. We're not catching it here, we're tracking it so we should care if it is fatal (Also please don't forget to update the description).